### DISCUSSION TITLE
If I get in contributors.

### DISCUSSION THREAD

**Discussion by itsimpeccable** [ScratchAddons/ScratchAddons#1130] [Discussion]
My name is ezxa on scratch, can you guys add me as ezxa if I will show up in Contributors?

**Comment by WorldLanguages** [comment]
Unfortunately I think we're only adding GitHub users.

**Reply by itsimpeccable** [reply]
Oh.

**Reply by Marc92020** [reply]
but we can add you as doruksarp and your github account

**Comment by cobaltt7** [comment]
What did you contribute?
Go [here](https://github.com/ScratchAddons/contributors/issues/12) and type:
> @all-contributors add @doruksarp for contributions

Replace "contributions" with what you helped with. See [this page](https://allcontributors.org/docs/en/emoji-key) to see available contribution types.

**Reply by cobaltt7** [reply]
wait i just noticed https://github.com/ScratchAddons/contributors/issues/12#issuecomment-752233952

guess this can be closed

**Reply by Marc92020** [reply]
someone close this

**Reply by cobaltt7** [reply]
@Marc92020 you can't close discussions, i didn't know that before
